.github/workflows/ci-cd.yaml

```diff
@@ -0,0 +1,72 @@
+---
+name: CI Build
+on:
+  schedule:
+    - cron: '28 0 * * 1'
+  push:
+    branches: ['master']
+  pull_request:
+    branches: ['master']
+
+jobs:
+  terraform:
+    name: "Terraform"
+    runs-on: ubuntu-latest
+    steps:
+      - name: "Checkout ${{ github.ref }} ( ${{ github.sha }} )"
+        uses: actions/checkout@v2
+
+      - name: "Setup Google SDK"
+        uses: GoogleCloudPlatform/github-actions/setup-gcloud@master
+        if: ${{ github.event_name == 'push' }}
+        with:
+          version: '307.0.0'
+          project_id: ${{ secrets.GCP_PROJECT_ID }}
+          service_account_key: ${{ secrets.GCP_SA_KEY }}
+          export_default_credentials: true
+
+      - run: gcloud info
+        if: ${{ github.event_name == 'push' }}
+
+      - name: "Setup Terraaform"
+        uses: hashicorp/setup-terraform@v1
+        with:
+          terraform_version: 0.12.25
+
+      - name: Terraform Init
+        run: terraform init
```

Comment:
```suggestion
        run: terraform init -input=false
```

Reply:
Why is it relevant in this case?

Reply:
Documentation says about `-input=false`
> indicates that Terraform should not attempt to prompt for input, and instead expect all necessary values to be provided by either configuration files or the command line

in case of failing job error message would be more informative

Reply:
Is is not covered by `TF_IN_AUTOMATION` env variable?